### PR TITLE
[jasmine] Support Jasmine 3.5

### PR DESCRIPTION
**Pull request by simondel**
- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [ ] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [ ] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/jasmine/jasmine/blob/master/release_notes/3.5.0.md
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.
- [x] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`. If for reason the any rule need to be disabled, disable it for that line using `// tslint:disable-next-line [ruleName]` and not for whole package so that the need for disabling can be reviewed.

### CODE REVIEW

File: types/jasmine/jasmine-tests.ts [hunk 1]
```diff
@@ -115,6 +115,18 @@ describe("Included matchers:", () => {
         expect(foo).not.toBeFalsy();
     });
 
+    it("The 'toBeTrue' matcher is for matching with true", () => {
+        expect(true).toBeTrue();
+        expect(false).not.toBeTrue();
+        expect({}).not.toBeTrue();
+    });
+
+    it("The 'toBeTrue' matcher is for matching with true", () => {
```
Comment (djungowski) [inline types/jasmine/jasmine-tests.ts]: Should be `"The 'toBeFalse' matcher is for matching with false"`

**Review by djungowski — CHANGES_REQUESTED** [review]
There's one c&p leftover, please fix that.

Apart from that for some reason there are a lot of whitespace changes. Have you run `npm run lint jasmine` to check, that these are not a problem?

File: types/jasmine/jasmine-tests.ts (hunk 1 above)
Comment (simondel) [reply types/jasmine/jasmine-tests.ts]: Nice catch! I've fixed it.

**Comment by simondel** [comment]
@djungowski The whitespace changes are not an issue:
```
 C:\dev\DefinitelyTyped> npm run lint jasmine

> definitely-typed@0.0.3 lint C:\dev\DefinitelyTyped
> dtslint types "jasmine"
```

**Review by djungowski — APPROVED** [review]

**Pull request merged**